dashboard/config/videos.csv

```diff
@@ -104,11 +104,12 @@ tutorial_puzzle_challenge			_l68chtl1co	http://videos.code.org/2014/tutorial-puz
 infinity_playlab_intro			vQGOfhsb76o	http://videos.code.org/2015/infinityplaylab/infinity-playlab-intro.mp4
 infinity_playlab_events			HcljJx_0Elw	http://videos.code.org/2015/infinityplaylab/infinity-playlab-events.mp4
 infinity_playlab_repeat_forever			Ldj1-kEa9nA	http://videos.code.org/2015/infinityplaylab/infinity-playlab-repeat-forever.mp4
+csp_applab_functions			yPWQfa4CHbw	https://videos.code.org/2015/csp/applab/functions.mp4
```

Comment:
Every other videos.code.org link in this file uses `http://` as the protocol.  Maybe we have to do the same? Maybe @wjordan can shed some light on this.